sky/packages/sky/lib/src/rendering/proxy_box.dart

```diff
@@ -389,6 +441,11 @@ class RenderClipRect extends RenderProxyBox {
   }
 }
 
+/// Clips its child using a rounded rectangle
+///
+/// Creates a rounded rectangle from its layout dimensions and the given x and
+/// y radius values and prevents its child from painting outside that rounded
+/// rectangle.
```

Comment:
Should probably say what happens for negative radii and radii that are more than half the largest extent.